components/html-block/demo/html-block.html

```diff
@@ -42,7 +42,7 @@
 
 		</script>
 		<script>
-			document.getElementsByTagName('html')[0].dataset.mathjaxContext = JSON.stringify({ renderLatex: window.location.search.indexOf('latex=true') !== -1 });
+			document.getElementsByTagName('html')[0].dataset.mathjaxContext = JSON.stringify({ outputScale: 1.1, renderLatex: window.location.search.indexOf('latex=true') !== -1 });
```

Comment:
This just makes it very easy to see that scaling is happening - `font-size` will be set on the various rendered math elements if scaling is working correctly.